x-pack/plugins/security/index.js

```diff
@@ -189,7 +189,7 @@ export const security = (kibana) => new kibana.Plugin({
       return new savedObjects.SavedObjectsClient(callWithRequestRepository);
     });
 
-    savedObjects.addScopedSavedObjectsClientWrapperFactory(Number.MIN_SAFE_INTEGER, ({ client, request }) => {
+    savedObjects.addScopedSavedObjectsClientWrapperFactory(Number.MAX_SAFE_INTEGER - 1, ({ client, request }) => {
```

Comment:
Swapping the order of SOC wrappers so that Spaces runs before Security. Previously, security ran before spaces, but this forced the security wrapper to authorize based on the request's space id.

By swapping the order of these wrappers, the Spaces wrapper can now change which space id (`namespace`) the Security plugin authorizes against. We never had a need for this before, but the copy-to-space functionality needs to operate across spaces, so there we go.